### DISCUSSION TITLE
GUI Color Panel

### DISCUSSION THREAD

**Discussion by benegeek** [linuxmint/discussions#393] [Ideas]
Why don’t you add a GUI color panel similar to this, that can be used to change intensity of red, green, blue (for all brightness, contrast and gamma), besides hue and saturation? It even allows you to create multiple color profiles:

![GUI](https://github.com/linuxmint/discussions/assets/166447959/285f811b-2976-4baa-95fa-0b7df8276040)

**Comment by Jopp-gh** [comment]
And your monitor doesn't come with a control panel by itself? 
This is hardware related stuff. 

Note, profile-critical apps have color profile management build-in like Gimp and other apps dealing with images. 

Otherwise, color profiling makes no sense - Default color profile settings are "usually" the best option for a monitor. 

To write (free) code is nothing easy or quick and requires context related knowledge and - time.  You could try and find out  yourself, if you can't wait.

**Reply by benegeek** [reply]
I use a laptop, they don't have a control panel by itself. For example I can't change the saturation level, and the standard level is usually exagerated.

**Reply by Jopp-gh** [reply]
I see!
The only way to modify your current monitor  settings is to find some 3rd party software - check out GitHub if you can't find in the official software center.
